pandas/core/generic.py

```diff
@@ -8518,24 +8518,198 @@ def _doc_parms(cls):
 """
 
 _cnum_doc = """
+Return %(desc)s over a DataFrame or Series axis.
+
+Returns a DataFrame or Series of the same size containing the %(desc)s.
 
 Parameters
 ----------
-axis : %(axis_descr)s
+axis : {0 or 'index', 1 or 'columns'}, default 0
 skipna : boolean, default True
     Exclude NA/null values. If an entire row/column is NA, the result
-    will be NA
+    will be NA.
+*args : default None
+**kwargs : default None
+    Additional keywords have no effect but might be accepted for
+    compatibility with NumPy.
 
 Returns
 -------
-%(outname)s : %(name1)s\n
-
-
+%(outname)s : %(name1)s or %(name2)s\n
+%(examples)s
 See also
 --------
-pandas.core.window.Expanding.%(accum_func_name)s : Similar functionality
+core.window.Expanding.%(accum_func_name)s : Similar functionality
     but ignores ``NaN`` values.
+Series.%(outname)s : Return %(desc)s over Series axis.
+DataFrame.cummax : Return cumulative maximum over DataFrame axis.
```

Comment:
Is it a good idea to also add: `DataFrame.sum` as a relevant method in the 'See also' section?

Reply:
Yes, that might be a good idea, if you can automatically make it link in `DataFrame.cumsum` to `DataFrame.sum`, in `DataFrame.cummin` to `DataFrame.min`, etc (using the templating)